### PR TITLE
Facilitation of Collaborative Work

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
This PR eases collaborative work:
- [x] Any user (not necessarily tester) can create organizations
- [x] Only study owners can edit inputs access level
- [x] Add option to share service with Everyone

## Related issue number
related to ITISFoundation/osparc-issues#345

## Checklist

- [ ] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [ ] Unit tests for the changes exist
- [ ] Runs in the swarm
- [ ] Documentation reflects the changes
- [ ] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]

File: services/web/client/source/class/osparc/component/export/StudyPermissions.js [hunk 1]
```diff
@@ -66,12 +66,7 @@ qx.Class.define("osparc.component.export.StudyPermissions", {
     __studyData: null,
 
     _isUserOwner: function() {
```
Comment (sanderegg) [inline services/web/client/source/class/osparc/component/export/StudyPermissions.js:68]: do you actually still need the function?

**Review by sanderegg — APPROVED** [review]
:+1:

File: services/web/client/source/class/osparc/component/export/StudyPermissions.js (hunk 1 above)
Comment (odeimaiz) [reply services/web/client/source/class/osparc/component/export/StudyPermissions.js:68]: Nope, thanks

Comment (odeimaiz) [reply services/web/client/source/class/osparc/component/export/StudyPermissions.js:68]: Yes, it needs to override the base class (servicePermissions are checked differently)

File: services/web/client/source/class/osparc/component/export/Permissions.js [hunk 2]
```diff
@@ -31,6 +31,10 @@ qx.Class.define("osparc.component.export.Permissions", {
 
     this.__buildLayout();
 
+    this.__collaborators = {};
+    initCollabs.forEach(initCollab => {
+      this.__collaborators[initCollab["gid"]] = initCollab;
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/component/export/Permissions.js:36]: when you share with everyone? Do you consider the product?
I.e. if a user shares with everyone in osparc, I assume everyone using osparc should get the service
but if he/she shares it in s4l product, then I assume "everyone" refers to users USING s4l specifically

Comment (odeimaiz) [reply services/web/client/source/class/osparc/component/export/Permissions.js:36]: @sanderegg

Comment (sanderegg) [reply services/web/client/source/class/osparc/component/export/Permissions.js:36]: As explained earlier: currently the product constraint is applied through the services access rights.

So:
Everyone is everyone regardless of the products. BUT, the services are constrained by products, so indirectly if you share a study with everyone while browsing on osparc, and that study uses services not available in s4l product, then that study will not appear in s4l.

**Pull request merged**